explainer.md

```diff
@@ -0,0 +1,329 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Background: assistive technology and the accessibility tree](#background-assistive-technology-and-the-accessibility-tree)
+  - [Accessibility node properties](#accessibility-node-properties)
+- [Existing accessibility APIs](#existing-accessibility-apis)
+  - [Native platform accessibility APIs](#native-platform-accessibility-apis)
+  - [Gaps in the web platform's accessibility story](#gaps-in-the-web-platforms-accessibility-story)
+  - [Audience for the proposed API](#audience-for-the-proposed-api)
+- [The Accessibility Object Model](#the-accessibility-object-model)
+  - [Exploring the accessibility tree](#exploring-the-accessibility-tree)
+- [Use cases](#use-cases)
+  - [Use DOM node references instead of IDREFs](#use-dom-node-references-instead-of-idrefs)
+  - [Create virtual accessibility trees](#create-virtual-accessibility-trees)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Background: assistive technology and the accessibility tree
+
+Assistive technology, in this context, refers to a third party application
+which augments or replaces the existing UI for an application.
+One well-known example is a screen reader,
+which replaces the visual UI and pointer-based UI
+with an auditory output (speech and tones)
+and a keyboard and/or gesture-based input mechanism,
+completely separate from any audio output or keyboard or gesture input handled by the app.
+
+Many assistive technologies interact with a web page via accessibility APIs, such as
+[UIAutomation](https://msdn.microsoft.com/en-us/library/windows/desktop/ee684009.aspx)
+on Windows, or
+[NSAccessibility](https://developer.apple.com/library/mac/documentation/AppKit/Reference/NSAccessibility_Protocol_Reference/)
+on OS X.
+These APIs allow an application to expose a tree of objects representing the application's interface,
+typically with the root node representing the application window,
+with various levels of grouping node descendants down to individual interactive elements.
+This is referred to as the **accessibility tree**.
+
+An assistive technology user interacts with the application almost exclusively via this API,
+as the assistive technology uses it both to create the alternative interface,
+and to route user interaction events triggered by the user's commands to the assistive technology.
+
+![Flow from application UI to accessibility tree to assistive technology to user](images/a11y-tree.jpg)
+
+For example, a [VoiceOver](https://www.apple.com/voiceover/info/guide/) user might press the key combination
+"Control Option Spacebar" to indicate that they wish to click the element which the screen reader is currently visiting.
+These keypresses would never be passed to the application,
+but would be interpreted by the screen reader,
+which would then call the `accessibilityPerformPress()` function on the element via the API.
```

Comment:
before you mentioned UIAutomation and NSAccessibility. is the `accessibilityPerformPress()` method available in both of these? Or is that a method on accessibility tree that blink builds before transforming it into the platform specific objects?